src/Integration.Vsix/SonarLintIntegrationPackage.cs

```diff
@@ -18,7 +18,7 @@ namespace SonarLint.VisualStudio.Integration.Vsix
     [ProvideAutoLoad(CommonGuids.PackageActivation)]
     [ProvideOptionPage(typeof(OptionsPage), OptionsPage.CategoryName, OptionsPage.PageName, 901, 902, false, 903)]
     [ProvideUIContextRule(CommonGuids.PackageActivation, "SonarLintIntegrationPackageActivation",
-         "Building & (HasCSProj | HasVBProj)",
+         "(HasCSProj | HasVBProj)",
         new string[] { "Building",
                        "HasCSProj",
                        "HasVBProj" },
```

Comment:
Remove both the `"Building"` and `VSConstants.UICONTEXT.SolutionBuilding_string` elements from these arrays. They are not needed since you've removed the `Building &` part of the above expression.